packages/vsce/src/webviews/resource-inspector-panel/utils/hyperlinkUtils.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import * as React from "react";
+import { IResourceProfileNameInfo } from "@zowe/cics-for-zowe-explorer-api";
+import * as vscode from "../../common/vscode";
```

Comment:
Import this directly and then you don't have to use `any`
```suggestion
import { postVscMessage } from "../../common/vscode";
```